state/stateConsensus.go

```diff
@@ -1370,42 +1371,21 @@ func (s *State) FollowerExecuteDataResponse(m interfaces.IMsg) {
 
 	switch msg.DataType {
 	case 1: // Data is an entryBlock
-		eblock, ok := msg.DataObject.(interfaces.IEntryBlock)
+		s.Logger.WithFields(logrus.Fields{
```

Comment:
logrus is a full featured logging package, but comes with very significant performance issues.
http://hackemist.com/logbench/

Clay's logging is very lightweight and is designed to feed into scripts that allow tracking of messages across processes.  Because we end up doing massive amounts of debugging of simulated networks in factomd, this is a huge advantage for development.

For low frequency system logging and exception processing, logrus is fine, but inside processing loops logrus should be avoided.